models/payment/src/index.ts

```diff
@@ -0,0 +1,5 @@
+// Export types
+export * from './payment-method-info/types';
+
+// Export models
+export * as Payment from './payment-method-info';
```

Comment:
I just noticed this should almost definitely be 
```suggestion
export * as PaymentMethodInfo from './payment-method-info';
```